testnets/celestiatestnet3/versions.json

```diff
@@ -21,12 +21,19 @@
     {
       "name": "v2",
       "recommended_version": "v2.1.2",
+      "compatible_versions": ["v2.0.0", "v2.1.0", "v2.1.1", "v2.1.2"]
+    },
+    {
+      "name": "v3",
+      "recommended_version": "v3.3.1-mocha",
       "compatible_versions": [
-        "v2.0.0",
-        "v2.1.0",
-        "v2.1.1",
-        "v2.1.2"
+        "v3.0.0-mocha",
+        "v3.0.1-mocha",
+        "v3.0.2-mocha",
+        "v3.1.1-mocha",
+        "v3.2.0-mocha",
+        "v3.3.0-mocha"
```

Comment:
compatible_versions is still missing "v3.3.1-mocha", so there is still a mismatch between this info in celestiatestnet3::chain.json::codebase vs here.

Reply:
thank you, Jeremy.